content/tutorial/tutorial.md

```diff
@@ -1145,11 +1152,11 @@ Next, we'll define the `jumpTo` method in Game to update that `stepNumber`. We a
   }
 ```
 
-We will now make a few changes to the Game's `handleClick` method which fires when you click on a square.
+अब हम गेम के 'handleClick' पद्धति में कुछ बदलाव करेंगे, जब आप एक square पर क्लिक करते हैं।
 
-The `stepNumber` state we've added reflects the move displayed to the user now. After we make a new move, we need to update `stepNumber` by adding `stepNumber: history.length` as part of the `this.setState` argument. This ensures we don't get stuck showing the same move after a new one has been made.
+हमारे द्वारा जोड़ा गया `stepNumber` state अब उपयोगकर्ता को प्रदर्शित की गई चाल को दर्शाता है। एक नई चाल बनाने के बाद, हमें `step.Number: history.length` को `this.setState` तर्क के भाग के रूप में जोड़कर `stepNumber` को अपडेट करने की आवश्यकता है। यह सुनिश्चित करता है कि हम एक नए कदम के बाद एक ही चाल दिखाने में फंस न जाएं।
```

Comment:
एक नई चाल chalne ke baad

Reply:
हमें `step.Number: history.length` को `this.setState` तर्क के भाग के रूप में जोड़कर `stepNumber` को अपडेट करने की आवश्यकता है। -> revisit

Reply:
एक नए कदम के बाद firse woi चाल दिखाने में फंस न जाएं।

Reply:
@arshadkazmi42 you missed this one

Reply:
> हमें `step.Number: history.length` को `this.setState` तर्क के भाग के रूप में जोड़कर `stepNumber` को अपडेट करने की आवश्यकता है। -> revisit

```
हमें `stepNumber: history.length` को `this.setState` के आर्गुमेंट में ऐड करके `stepNumber` को  अपडेट करना है
```

How about this?

Reply:
sounds good